.github/workflows/commands.yml

```diff
@@ -1,22 +1,22 @@
-name: Repo Commands
-
-on:
-  issue_comment:        # Handle comment commands
-    types: [created]
-  pull_request_target:  # Handle renamed PRs
```

Comment:
@extremeheat what is correct ?

Reply:
"pull_request_target" is better as "pull_request" can only be triggered  by people with write access to the base repo. This is just for handling renamed release PRs (so it can update version inside PR), but since that can only be done by people with issue permissions on the repo it's just a minor change.